ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-main__okd.yaml

```diff
@@ -1,10 +1,10 @@
 base_images:
   base:
-    name: "4.14"
+    name: "4.15"
```

Comment:
Please do this in a separate PR since it's focusing on main

Reply:
Removed it from the okd.yaml file.